js/components/controlBar.js

```diff
@@ -202,7 +205,6 @@ var ControlBar = React.createClass({
           typeof this.qualityBtnElement.focus === 'function') {
         this.qualityBtnElement.focus();
       }
-      this.qualityMenuOpenedWithKeyboard = false;
```

Comment:
Shouldn't we keep this here ?

Reply:
Nope, this was moved to `toggleQualityPopover` which is called below.